libs/template/templates/default/template/__preamble.tmpl

```diff
@@ -20,12 +20,7 @@ This file only contains template directives; it is skipped for the actual output
   {{skip "{{.project_name}}/src/{{.project_name}}"}}
 {{end}}
 
-{{if or $python_package $notebook_job $pipeline}}
-  # Remove .gitkeep files for a non-empty project
-  {{skip "{{.project_name}}/src/.gitkeep"}}
-  {{skip "{{.project_name}}/resources/.gitkeep"}}
-{{else}}
-  # Fully empty project, even remove the sample job
```

Comment:
What's the reason for removing gitkeeps?

It's not possible to test this without custom acceptance tests.

Reply:
Ultimately, this is me being perfectionistic! I don't want those files in the output. Could you take another look, @pietern?

Reply:
That's fine, but keep in mind that this is not covered in the acceptance tests.

You could add a dedicated one to check that the directories are kept around. Or if you're comfortable with the unit test alone, that's fine too.